docs/blog/2022-12-12-Building-OKD-payload.md

```diff
@@ -0,0 +1,265 @@
+    testRHCOS:::RHCOS -. against .- tis
+
+    PR[fa:fa-github PullRequest]-- submitted -->PreSubSCOS{{fa:fa-gears fa:fa-prow PreSubmitJob / OKD}};
+    PreSubSCOS:::SCOS-- builds -->tis[(temporary IS)];
+    PreSubSCOS-- runs -->testSCOS{{Platform tests}};
+    testSCOS:::SCOS -. against .- tis
+    
+    PR-- merged -->PostSubRHCOS{{fa:fa-gears PostSubmitJob / OCP CI}};
+    PostSubRHCOS:::RHCOS-- promotes -->isocp[(OCP CI IS 4.x)];
+    PostSubRHCOS -. from .- tis
+
+    PR-- merged -->PostSubSCOS{{fa:fa-gears PostSubmitJob / OKD}};
+    PostSubSCOS:::SCOS-- promotes -->isokd[(OKD IS scos-4.x)]:::SCOS;
+    PostSubSCOS -. from .- tis
+    
+     classDef RHCOS fill:;
+     classDef SCOS fill:#1e97fa;
+```
+
+The built images land directly in the `scos-${MAJOR}-${MINOR}` ImageStream. 
+
+That is why there's no need for OCP's CI release controller to mirror these images from the CI ImageStream: 
+During the PostSubmit phase, images are already getting built in parallel for OCP, OKD/FCOS and OKD/SCOS and pushed, respectively to `ocp/$MAJOR.$MINOR`, `origin/$MAJOR.$MINOR`, `origin/scos-$MAJOR.$MINOR`
+
+
+
+### OKD release builds
+
+Now the ImageStream `scos-$MAJOR.$MINOR` is getting populated by payload images. With every new image tag, the [release controller for OKD/SCOS](https://github.com/openshift/release/blob/master/core-services/release-controller/_releases/release-okd-scos-4.13.json) will  build a release image.
+
+The ReleaseController ensures that OpenShift update payload images (aka release images) are created whenever an ImageStream representing the images in a release is updated.  
+
+Thanks to the annotation `release.openshift.io/config` on the `scos-${MAJOR}-{MINOR}` ImageStream, the controller will:
+1. Create a tag in the `scos-${MAJOR}-{MINOR}` ImageStream that uses the release name + current timestamp.
+2. Mirror all of the tags in the input ImageStream so that they can't be pruned.
+3. Launch a job in the job namespace to invoke `oc adm release new` from the mirror pointing to the release tag we created in step 1.
+4. If the job succeeds in pushing the tag, it sets an annotation on that tag `release.openshift.io/phase = "Ready"`, indicating that the release can be used by other steps. And that's how a new release appears in https://origin-release.ci.openshift.org/#4.13.0-0.okd-scos
+5. The release state switches to "Verified" when the [verification end-to-end test job](https://github.com/openshift/release/blob/master/core-services/release-controller/_releases/release-okd-scos-4.13.json#L20)  succeeds.
+
+
+## Building the Tekton way :motorway:
+
+Building with Prow has the advantage of being driven by new code being pushed to payload components, thus building fresh releases as the code of [github.com/openshift](https://github.com/openshift) evolves.
+
+The problem is that Prow, along with all the clusters involved with it, the image streams, etc. are not accessible to the OKD community outside of RedHat.
```

Comment:
We should add smth like "users also may want a custom OKD payload built privately"